src/components/listings-grid.js

```diff
@@ -23,7 +23,8 @@ class ListingsGrid extends Component {
   render() {
     const { listingsPerPage } = this.state
     const { contractFound, listingIds, hideList } = this.props
-    const pinnedListingIds = [0, 1, 2, 3, 4]
+    //const pinnedListingIds = [0, 1, 2, 3, 4]
```

Comment:
Those hardcoded pinnedListingIds are used for "featuring" listings.
We'll need to replace this with a programmatic way (we discussed having some sort of admin UI allowing Origin employees to feature/unfeature listings).
But for now, we should probably keep the list ?

Reply:
Yeah this is a mistake from my part. These ones shouldn't be commented out. Thanks for the catch